client/src/components/GroupChatsList.tsx

```diff
@@ -0,0 +1,32 @@
+// src/components/GroupChatList.tsx
+import React from 'react'
+import ChatTile from './ChatTile'
+import Search from './Search' 
+import styles from './GroupChatsList.module.css'
+type LastMessageStatus = 'read' | 'unread'
+interface ChatData {
+    chatName: string
+    profilePicture: string
+    lastMessage: string
+    lastMessageSenderName: string
+    lastMessageStatus: LastMessageStatus
+}
+
+interface Props {
+    chats: ChatData[]
+}
+
+const GroupChatsList: React.FC<Props> = ({ chats }) => {
+    return (
+        <div className={styles.groupChatsList}>
+            {chats.map((chat, index) => (
+                <ChatTile key={index} {...chat} />
```

Comment:
Nie jest to dobra praktyka, by kluczem był indeks z mapy. Może nam się zmienić kolejność czatów, a my tego nie przewidzimy. 

Zaciągnąłbym jakąś libkę co generuje UUIDy i po prostu tu odpalał uuid()